compact/ecal/barrel_interlayers.xml

```diff
@@ -65,10 +66,10 @@
                      (EcalBarrel_ImagingLayerThickness + EcalBarrel_ScFiLayerThickness +
                       EcalBarrel_SpaceBetween)))"/>
     <comment>
-      Adjusting size of the ScFi back chunk to match number of imiaging layers
+      Adjusting size of the ScFi back chunk to match number of imaging layers
       and 20 radiation lengths in total
```

Comment:
Is this still correct with the increased thickness?

Reply:
It is correct. The total thickness of the SciFi calorimeter haven't increased. It is now 16*16*1.22mm = 31.23 cm mm and before it was 20*13*1.22mm = 31.7 cm. This is about 20 X0. See also my reply to the comment above.

Reply:
Sorry, the comment from @veprbl didn't show up on the android interface since it is (supposedly) 'outdated'.